src/signupTutor.js

```diff
@@ -64,89 +64,17 @@ class SignupTutor extends React.Component {
-                </div>
-                <div className="row col-xs-12">
-                    <input
-                    type="text"
-                    className={this.state.usernameValidation + " input-lg col-xs-10 col-xs-offset-1"}
-                    placeholder="Create A Username"
-                    value={this.state.username}
-                    onChange={this.handleUsernameChange} />
-                    <HelpBlock className={this.state.usernameValidation === 'input-error' ? 'show-error' : 'hide-error'}>Username must be at least 4 characters.</HelpBlock>
-                </div>
-                <div className="row col-xs-12">
-                    <input
-                    type="password"
-                    className={this.state.passwordValidation + " input-lg col-xs-10 col-xs-offset-1"}
-                    placeholder="Create A Password"
-                    value={this.state.password}
-                    onChange={this.handlePasswordChange} />
-                    <HelpBlock className={this.state.passwordValidation === 'input-error' ? 'show-error' : 'hide-error'}>Password must be at least 6 characters.</HelpBlock>
-                </div>
-                <div className="row col-xs-12">
-                    <input
-                    type="password"
-                    className={this.state.confirmPasswordValidation + " input-lg col-xs-10 col-xs-offset-1"}
-                    placeholder="Confirm Password"
-                    value={this.state.confirmPassword}
-                    onChange={this.checkConfirmPassword}/>
-                    <HelpBlock className={this.state.confirmPasswordValidation === 'input-error' ? 'show-error' : 'hide-error'}>Passwords must match.</HelpBlock>
-                </div>
-                <div className="row col-xs-12">
-                    <input
-                    type="text"
-                    className="input-lg col-xs-10 col-xs-offset-1"
-                    placeholder="Georgia Tech Email"
-                    value={this.state.email}
-                    onChange={this.handleEmailChange}/>
-                </div>
-                <div className="row col-xs-12">
-                    <input
-                    type="text"
-                    className="input-lg col-xs-10 col-xs-offset-1"
-                    placeholder="Confirm Email"/>
-                </div>
-                <div className="row col-xs-12">
-                    <input
-                    className="signup-button btn btn-lg btn-default col-xs-10 col-xs-offset-1"
-                    type="submit"
-                    value="SUBMIT" />
-                </div>
-                <div className="row col-xs-12">
-                    <h5 className="signup-dialogue">Already have an account? <a className="signup-anchor" href='/home/login'>Click here to log in!</a></h5>
-                </div>
-            </form>
-        </div>
-        );
+    if (currentConfirmPassword === this.state.password) {
```

Comment:
Lol idk what happened here, but it seems like we lost of bunch of stuff? Is this file not used anymore maybe? I remember my branch `signup-validation` has a file called SignUpTutor (with caps)

Reply:
Who did the sign up validation?